volumes/ecocounter/views/create-view-recent-outages.sql

```diff
@@ -0,0 +1,73 @@
+CREATE OR REPLACE VIEW ecocounter.recent_outages AS (
```

Comment:
can we rename to sth like `last_week_outages` or sth similar?

Reply:
month**

Reply:
even more generic: `identify_outages`